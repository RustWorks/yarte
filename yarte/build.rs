#[path = "src/recompile.rs"]
mod recompile;

fn main() {
    recompile::when_changed();
}
