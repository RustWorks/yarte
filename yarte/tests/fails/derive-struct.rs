use yarte::Template;

#[derive(Template)]
#[template(src = "")]
enum Test {
    F
}

fn main() {}
