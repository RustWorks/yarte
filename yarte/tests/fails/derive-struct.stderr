error: need a `struc`
 --> $DIR/derive-struct.rs:4:1
  |
4 | / #[template(src = "")]
5 | | enum Test {
6 | |     F
7 | | }
  | |_^
